src/SamplesApp/UITests.Shared/Windows_UI_Xaml/DragAndDrop/DragDrop_TestPage.xaml.cs

```diff
@@ -0,0 +1,393 @@
+		/// </summary>
+		private void DropBorder_DragOver(object sender, global::Windows.UI.Xaml.DragEventArgs e)
+		{
+			this.ShowDragDropDetails(nameof(global::Windows.UI.Xaml.UIElement.DragOver), e);
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when an item is dropped on the <see cref="DropBorder"/>.
+		/// </summary>
+		private void DropBorder_Drop(object sender, global::Windows.UI.Xaml.DragEventArgs e)
+		{
+			this.ShowDragDropDetails(nameof(global::Windows.UI.Xaml.UIElement.Drop), e);
+			this.ShowDropDetails(e);
+
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when an internal drag starts from a known source.
+		/// </summary>
+		private void DragSource_DragStarting(global::Windows.UI.Xaml.UIElement sender, DragStartingEventArgs args)
+		{
+			string availableStdFormats = string.Empty;
+			string requestedOperation = DataPackageOperation.Copy.ToString();
+
+			args.Data.RequestedOperation = DataPackageOperation.Copy;
+			args.AllowedOperations = DataPackageOperation.Copy | DataPackageOperation.Link | DataPackageOperation.Move;
+
+			if (object.ReferenceEquals(sender, this.DragTextSourceTextBlock))
+			{
+				var sourceText = ((TextBlock)sender).Text;
+				args.Data.SetText(sourceText);
+
+				availableStdFormats = nameof(StandardDataFormats.Text);
+			}
+			else if (object.ReferenceEquals(sender, this.DragImageSourceImage))
+			{
+				var imageUri = RandomAccessStreamReference.CreateFromUri(new Uri(@"https://upload.wikimedia.org/wikipedia/commons/thumb/d/da/Rain_over_Beinn_Eich%2C_Luss_Hills%2C_Scotland.jpg/2560px-Rain_over_Beinn_Eich%2C_Luss_Hills%2C_Scotland.jpg"));
+				args.Data.SetBitmap(imageUri);
+
+				availableStdFormats = nameof(StandardDataFormats.Bitmap);
+			}
+			else if (object.ReferenceEquals(sender, this.DragWebLinkSourceTextBlock))
+			{
+				var link = ((TextBlock)sender).Text;
+				args.Data.SetWebLink(new Uri(link));
+
+				availableStdFormats = nameof(StandardDataFormats.WebLink);
+			}
+
+			var eventDetails = new EventDetails()
+			{
+				EventBackground = this.DragStartingBrush,
+				EventName = nameof(global::Windows.UI.Xaml.UIElement.DragStarting),
+				AvailableFormats = string.Empty,
+				AvailableStdFormats = availableStdFormats,
+				RequestedOperation = requestedOperation,
+				Timestamp = DateTimeOffset.Now.ToString("HH:mm:ss.fff")
+			};
+
+			this.AddToDragDropEvents(eventDetails);
+
+			return;
+		}
+
+		/// <summary>
+		/// Event handler for when a drop operation is completed.
+		/// </summary>
+		private void DragSource_DropCompleted(global::Windows.UI.Xaml.UIElement sender, DropCompletedEventArgs args)
+		{
+			var eventDetails = new EventDetails()
+			{
+				EventBackground = this.DropCompletedBrush,
+				EventName = nameof(UIElement.DropCompleted),
+				AvailableFormats = string.Empty,
+				AvailableStdFormats = string.Empty,
+				RequestedOperation = string.Empty,
+				Timestamp = DateTimeOffset.Now.ToString("HH:mm:ss.fff")
```

Comment:
Codacy found an issue: [Use the overload that takes a 'CultureInfo' or 'IFormatProvider' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6333952)